Algorithms/String-Algorithm/Isomorphic.js

```diff
@@ -0,0 +1,21 @@
+s = "abc"
+t = "aaa"
+
+var isIsomorphic = function(s, t) {
+    if (s.length != t.length)
+        return false
+    let m = new Map()
+    for (let i = 0; i < s.length; i++) {
+        if (!m.has(s[i]))
+            m.set(s[i], t[i])
+        else {
+            
+            if (m.get(s[i]) != t[i]) {
+                
+                return false
+            }
+        }
+    }
+    return new Set([...m.values()]).size == m.size
+    
+};
```

Comment:
//DYNAMICALLY TAKING STRINGS 
var input = []; // initialise an empty array
var temp = '';
do {
    temp = prompt("Enter a number. Press cancel or leave empty to finish.");
    if (temp === "" || temp === null) {
        break;
    } else {
        input.push(temp);  
    }
} while (1);